packages/calypso-e2e/src/lib/pages/gutenberg-editor-page.ts

```diff
@@ -302,4 +307,20 @@ export class GutenbergEditorPage {
 		await Promise.all( [ this.page.waitForNavigation(), frame.click( selectors.viewButton ) ] );
 		await this.page.waitForLoadState( 'networkidle' );
 	}
+
+	/**
+	 * Opens the Block Editor sidebar (left hand side).
+	 */
+	async toggleSidebar(): Promise< void > {
```

Comment:
Toggles the sidebar by clicking on the `W`.

Note, on mobile this cannot be used unless in landscape mode - see https://github.com/Automattic/wp-calypso/issues/51162.